### DISCUSSION TITLE
How to change font size for markup notes ?

### DISCUSSION THREAD

**Discussion by theelectricsoul** [freeplane/freeplane#997] [Help] [answered]
Hello

![image](https://user-images.githubusercontent.com/10260832/217790578-85ddb079-a948-4ed4-a86d-55f569edf3f3.png)

Thank you :)

**Comment by euu2021** [comment, marked as answer]
I think the solution is [changing the CSS for the notes](https://docs.freeplane.org/user-documentation/css-for-html-or-markdown.html#css-for-details-and-note).

**Reply by theelectricsoul** [reply]
ahhhh    so for note css I have to go through the styles....  nice :=
